src/opam_publish.ml

```diff
@@ -431,20 +431,33 @@ let update_mirror repo =
       git ["reset"; "origin/master"; "--hard"];
     )
 
-let repo_package_dir package =
+let add_trailing_slash filename =
+  if Filename.check_suffix filename "/"
+  then filename
+  else filename ^ "/"
+
+let repo_package_dir (repo_subdirectory : string option) package =
+  let repo_subdirectory : string =
+    match repo_subdirectory with
+    | None ->
+        ""
+    | Some repo_subdirectory ->
+        add_trailing_slash repo_subdirectory
+  in
```

Comment:
As the string is then given in argument to `OpamFilename.Dir.of_string`, and concatened using the operator `OpamFilename.Op.(/)`, there is no need to add this trailing slash

Reply:
Simplified, thanks.